cppframework/Foo.cpp

```diff
@@ -2,9 +2,6 @@
 #include <string>
 #include <iostream>
 
-#define EXPORT __attribute__((visibility("default")))
```

Comment:
Hm. Looks like while testing various compilation options I broke something. When I started from clean project and enabled testability it worked on CMake project. So to reiterate, I think we should use default visibility options from Xcode. If you still want to hide symbols then you should use XCODE_ATTRIBUTE_GCC_SYMBOLS_PRIVATE_EXTERN which is applicable to all kinds of symbols, not just C++, C and ObjC++.

Why do you actually want to hide those symbols? If it is for obfuscation then Archive is stripping all externally unused symbols anyways... If it is matter of controlling what client code can reference from framework then I'm not sure I see the point...